code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -148,7 +148,9 @@ Contains most of the procs that are called when a mob is attacked by something
 		visible_message("\red <B>[src] has been [pick(I.attack_verb)] in the [hit_area] with [I.name] by [user]!</B>", null, null, 5)
 	else
 		visible_message("\red <B>[src] has been attacked in the [hit_area] with [I.name] by [user]!</B>", null, null, 5)
-
+	
+	if(command_aura == "move") //Check for move aura; improves melee damage significantly.
+		I.force *= (1 + (command_aura_strength * 0.33)) // CO move orders temporarily double melee damage (guns are probably still better, but improves those PBs).
```

Comment:
`/mob/living/carbon/human/proc/attacked_by(var/obj/item/I, var/mob/living/user, var/def_zone)`
Are you sure you're on the right proc?

Do weapons forces routinely get recalculated? If not, then weapons used against humans will grow exponentially in force.